tests/0.2.4.1/post.yaml

```diff
@@ -118,18 +118,18 @@ tests:
             name: "michael"
             age: 38
             key: "michael:1234"
-        - device:
+          device:
             mobile: false
-        - team:
+          team:
             id: 3.5
       expected_data:
         - user:
             name: "michael"
             age: 38
             key: "michael:1234"
-        - device:
+          device:
             mobile: false
-        - team:
+          team:
             id: 3.5
```

Comment:
This was an array of three objects instead of a single object that had three nodes in it.